editor/blocks/list/index.js

```diff
@@ -4,22 +4,26 @@
 import './style.scss';
 
 const Editable = wp.blocks.Editable;
-const { html, prop } = wp.blocks.query;
+const { html, prop, query, parse } = wp.blocks.query;
+
+const parseAttrs = {
+	listType: prop( 'ol,ul', 'nodeName' ),
+	items: query(
+		'li',
+		{
+			value: ( node ) => {
```

Comment:
This change was made as part of the debugging process. It can definitely be reverted. We were tracking where this was called in the overall system.